app/models/setting/auth.rb

```diff
@@ -59,3 +59,4 @@ def validate_websockets_ssl_cert(record)
   end
 end
 
+require_dependency 'setting'
```

Comment:
It fixes a load error in the 'foreman' initialiser, which is loading these files early on and calling load_defaults.  Without also loading the main setting.rb, the superclass isn't loading properly and the super class method can't be found.